ember-link/src/services/link-manager.ts

```diff
@@ -8,11 +8,13 @@ import { BEHAVIOR, prevent } from '../-behavior';
 import { getOwner } from '../-owner';
 import Link from '../link';
 
+import type { RouteModel } from '../-models';
 import type { Behavior } from '../-behavior';
 import type { LinkParams } from '../-params';
 import type RouteInfo from '@ember/routing/route-info';
 import type RouterService from '@ember/routing/router-service';
 import type Transition from '@ember/routing/transition';
+import Owner from '@ember/owner';
```

Comment:
when I was testing this branch locally I noticed that the changes here were causing some linting errors

```
| [js] /--------/buschtoens/ember-link/ember-link/src/services/link-manager.ts
│ [js]    1:1  error  Run autofix to sort these imports!                                        simple-import-sort/imports
│ [js]   17:1  error  All imports in the declaration are only used as types. Use `import type`  @typescript-eslint/consistent-type-imports
│ [js]
│ [js] ✖ 2 problems (2 errors, 0 warnings)
│ [js]   2 errors and 0 warnings potentially fixable with the `--fix` option.
```